examples/pytorch/aim.py

```diff
@@ -1,3 +1,5 @@
+# pip install "lightly[timm]"
+
 # Note: The model and training settings do not follow the reference settings
 # from the paper. The settings are chosen such that the example can easily be
 # run on a small dataset with a single GPU.
```

Comment:
Let's add some context for the pip install command, otherwise it is unclear why it is there and looks a bit weird in the final notebook/examples:

![Screenshot 2024-08-09 at 09 30 24](https://github.com/user-attachments/assets/43a54213-09d3-4aaf-9ba3-eb90818b2ca2)

```suggestion
# This example requires the following dependencies to be installed:
# pip install "lightly[timm]"

# Note: The model and training settings do not follow the reference settings
# from the paper. The settings are chosen such that the example can easily be
# run on a small dataset with a single GPU.
```

Reply:
If I'd modified the examples, we'd have been left with extra markdown cells with the installation instructions, instead now in the generation script we add that statement to each notebook (and also drop the repeated pip install cmd)